frontend/create_ux_bundle.rst

```diff
@@ -0,0 +1,123 @@
+.. index::
+   single: Create a UX bundle
+
+Create a UX bundle
+==================
+
+.. tip::
+
+    Before reading this, you may want to have a look at
+    :doc:`Best Practices for Reusable Bundles </bundles/best_practices>`.
+
+Here are a few tricks to make your bundle install as a UX bundle.
+
+composer.json file
+------------------
+
+Your ``composer.json`` file must have the ``symfony-ux`` keyword:
+
+.. code-block:: json
+
+    {
+        "keywords": ["symfony-ux"]
+    }
+
+Assets location
+---------------
+
+Your assets must be located in one of the following directories, with a ``package.json`` file so Flex can handle it
+during install/update:
+
+* ``/assets`` (recommended)
+* ``/Resources/assets``
+* ``/src/Resources/assets``
+
+package.json file
+-----------------
+
+Your ``package.json`` file must contain a ``symfony`` config with controllers defined, and also add required packages
+to the ``peerDependencies``:
+
+.. code-block:: json
+
+    {
+        "name": "@acme/feature",
+        "version": "1.0.0",
+        "symfony": {
+            "controllers": {
+                "slug": {
+                    "main": "dist/controller.js",
+                    "webpackMode": "eager",
+                    "fetch": "eager",
+                    "enabled": true,
+                    "autoimport": {
```

Comment:
We should probably have a section documenting all the available options for a controller

Reply:
Not a bad idea to have a list explaining the keys below.

And actually, thanks to this comment, I caught one thing: remove `webpackMode`. That's present in existing UX packages as a legacy thing.

Reply:
Options have been added, and `webpackMode` is removed. :)